src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/Wallet.java

```diff
@@ -1929,4 +2027,60 @@ AccountUpdateKey makeUncompressedKeyFormat(AccountUpdateKey updateKey) {
 
         return updateKey;
     }
+
+    /**
+     * Validate whether given migrationAccount is valid or not.
+     * @param migrationAccount An account to be migrated to KAS Wallet
+     * @throws IllegalArgumentException
+     */
+    private void validateMigrationAccount(MigrationAccount migrationAccount) throws IllegalArgumentException {
```

Comment:
Do we need to throw IllegalArgumentException? as i know, IllegalArgument exception is unchecked exception.

Reply:
@sirano11 
You are right. IllegalArgument class extends Runtime Exception class which is unchecked exception.
I'll remove un-necessary throws clause at method definition.

Thanks for your feedback.